15_tictactoe/go/io/io_test.go

```diff
@@ -6,17 +6,18 @@ import (
 
 func TestValid(t *testing.T) {
 	tests := []struct {
-		input string
-		want  bool
+		input     string
+		testSlice []int
```

Comment:
I would change the name to this, for something more descriptive.
```suggestion
		validChoices []int
```

Reply:
Makes sense. Added.